shared/src/ledger/eth_bridge/storage/eth_msgs.rs

```diff
@@ -0,0 +1,199 @@
+//! Functionality for accessing the `eth_msgs/...` subspace
+use crate::types::ethereum_events::EthereumEvent;
+use crate::types::hash::Hash;
+use crate::types::storage::Key;
+
+#[allow(missing_docs)]
+pub const PREFIX_KEY_SEGMENT: &str = "eth_msgs";
+
+/// Get the key prefix corresponding to where details of seen [`EthereumEvent`]s
+/// are stored
+pub fn prefix() -> Key {
+    super::prefix()
+        .push(&PREFIX_KEY_SEGMENT.to_owned())
+        .expect("should always be able to construct this key")
+}
+
+#[allow(missing_docs)]
+pub const BODY_KEY_SEGMENT: &str = "body";
+#[allow(missing_docs)]
+pub const SEEN_KEY_SEGMENT: &str = "seen";
+#[allow(missing_docs)]
+pub const SEEN_BY_KEY_SEGMENT: &str = "seen_by";
+#[allow(missing_docs)]
+pub const VOTING_POWER_KEY_SEGMENT: &str = "voting_power";
```

Comment:
nit: can the visibility of these items be restricted to something like `pub(crate)`? this way cargo won't complain about missing docs, either, I don't think

Reply:
maybe `pub` is not needed at all

Reply:
These were intended to be public but I'm not sure what the use case would be tbh. I was thinking an external tool or test crate that tried to read/write storage under the EthBridge account might use the segments. But they would probably just use the `eth_msgs::Keys` struct instead

Reply:
I've removed the `pub` from all these consts